### PR TITLE
Fix `loo_compare`

### PR DESCRIPTION
**Pull request by smith-garrett**
The minimal working example in #91 works now.

### CODE REVIEW

**Comment by devmotion** [comment]
IMO the better fix would be to replace `Not` with standard indexing.

File: src/ModelComparison.jl [hunk 1]
```diff
@@ -98,7 +98,8 @@ function loo_compare(
     )
     # Subtract the effective number of params and elpd ests; leave mcse+pareto_k the same
     @views base_case = pointwise_diffs[:, 1:3, 1]
-    @views @. pointwise_diffs[:, 1:3, Not(1)] = pointwise_diffs[:, 1:3, Not(1)] - base_case
+    lastidx = axes(pointwise_diffs, 3)[2:end]
```
Comment (devmotion) [inline src/ModelComparison.jl:101]: This is wrong in general:
```julia
julia> using OffsetArrays

julia> x = OffsetArray(1:7, -3:3)
1:7 with indices -3:3

julia> axes(x, 1)[2:end]
2:3

julia> axes(x, 1)
OffsetArrays.IdOffsetRange(values=-3:3, indices=-3:3)
```

File: src/ModelComparison.jl [hunk 2]
```diff
@@ -98,7 +98,8 @@ function loo_compare(
...
+    @views @. pointwise_diffs[:, 1:3, lastidx] = pointwise_diffs[:, 1:3, lastidx] - base_case
```
Comment (devmotion) [inline src/ModelComparison.jl:102]:
```suggestion
    @views pointwise_diffs[:, 1:3, (begin + 1):end] .-= base_case
```

File: src/ModelComparison.jl (hunk 1 above)
Comment (smith-garrett) [reply src/ModelComparison.jl:101]: Good catch. I haven't worked with OffsetArrays, so I'm not as familiar with other indexing systems.

File: src/ModelComparison.jl (hunk 2 above)
Comment (smith-garrett) [reply src/ModelComparison.jl:102]: This is much cleaner!

**Review by devmotion — APPROVED** [review]

**Comment by devmotion** [comment]
Thank you!

**Pull request merged**